TOOLS/ce/src/main/scala/org.opalj.ce/ConfigEntry.scala

```diff
@@ -0,0 +1,58 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+/**
+ * Stores a value inside the structure of the configNode
+ * @param value is the value stored in the entry
+ * @param comment are all the comments associated with the value
+ */
+case class ConfigEntry(value: String, comment: Comment) extends ConfigNode {
+    /**
+     * Formats the entry into HTML code
+     * @param label required if the Entry is part of an object (Writes the key of the K,V Map there instead). Overrides the label property of the Comment object.
+     * @param HTMLHeadline accepts the HTML syntax of the Headline of the value. Can contain $ label and $ brief flags for filling with content
+     * @param HTMLContent accepts the HTML syntax of the content frame for the value. Must contains a $ content flag for correct rendering
+     * @return returns the Config Entry as HTML code
+     */
+    override def toHTML(label: String, HTMLHeadline: String, HTMLContent: String, sorted: Boolean): String = {
+        var HTMLString = ""
+        var head = label
+        var brief = this.comment.brief
+        if (this.comment.label.nonEmpty) head = this.comment.label
+        else if (head == "") head = value
+
+        // Write value into HTML code
+        var content = "<b>Value: </b><code>" + value.replace("<", "&lt").replace(">", "&gt") + "</code><br>"
+        content += this.comment.toHTML
+
+        // If there is no brief preview, put the value into it
+        if (comment.brief.isEmpty) {
+            brief = "<b>Value: </b><code>" + value.replace("<", "&lt").replace(">", "&gt") + "</code>"
```

Comment:
Do the `value.replace` only one, then reuse the result

Reply:
That is exactly what two chained .replace operations do.

Reply:
The point is that you do the exact same two operations already in line 26

Reply:
I see. The issue here is a different one. I will write another object that supplies another function to escape special characters into html format. Then, I can simply call the object to replace it.

Reply:
I am using the org.apache.commons library for escaping all special characters from the html, which practically is already used by the bi subproject.

Reply:
But the problem that you escape value twice here still remains.